lib/audit_test.pm

```diff
@@ -158,14 +157,23 @@ sub compare_run_log {
 #
 sub _parse_results_with_diff_baseline {
     my ($name, $result, $msg, $flag) = @_;
+    my $softfail_tests = {
+        s390x => {
+            ssh04 => 'Test case ssh04 fails in s390x is a known issue, see poo#99096',
+            iptables_SETSOCKOPT => 'bsc#1192324'
+        }
+    };
     if ($result eq 'PASS') {
         record_soft_failure($msg);
         $flag = 'softfail' if ($flag ne 'fail');
     }
     else {
-        if ($name eq 'ssh04' && is_s390x) {
-            record_soft_failure('Test case ssh04 fails in s390x is a known issue, see poo#99096');
-            return 'softfail';
+        my $arch = get_var('ARCH');
+        if ($softfail_tests->{$arch}) {
+            if (my $msg = $softfail_tests->{$arch}->{$name}) {
+                record_soft_failure($msg);
```

Comment:
Check https://openqa.suse.de/tests/7968051#step/ip_eb_tables/36:
Could you please add/show the test case name?

Reply:
same as lily's comment, it looks like it shows only.
```
# Soft Failure:
bsc#1192324
```